src/pipeline_v3/cli/management.py

```
@@ -550,80 +509,124 @@ def _resolve_sources(self, sources: List[str], recursive: bool = False, url_file
+                    ]
                     for match in matches:
                         match_path = Path(match)
-                        if match_path.is_file() and match_path.suffix.lower() in supported_extensions:
+                        if (
+                            match_path.is_file()
+                            and match_path.suffix.lower() in supported_extensions
+                        ):
                             resolved.append(match)
                 else:
                     print(f"Warning: No files found matching pattern: {source}")
             except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)